core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -485,4 +485,13 @@ package object config {
         "array in the sorter.")
       .intConf
       .createWithDefault(Integer.MAX_VALUE)
+
+  private[spark] val SHUFFLE_MAP_OUTPUT_STATISTICS_MULTITHREAD_THRESHOLD =
+    ConfigBuilder("spark.shuffle.mapOutputStatisticsMultithreadThreshold")
```

Comment:
`spark.shuffle.mapOutputStatistics.parallelAggregationThreshold`?

Reply:
Yes, it's better!